app/src/main/java/cm/aptoide/pt/home/apps/AppsManager.java

```diff
@@ -200,16 +224,16 @@ public Completable pauseUpdate(App app) {
         () -> installManager.stopInstallation(((UpdateApp) app).getMd5()));
   }
 
-  public Completable updateApp(App app) {
+  public Completable updateApp(App app, boolean isAppcUpdate) {
     String packageName = ((UpdateApp) app).getPackageName();
     return updatesManager.getUpdate(packageName)
         .flatMap(update -> {
-          Download value = downloadFactory.create(update);
+          Download value = downloadFactory.create(update, isAppcUpdate);
           return Observable.just(value);
         })
         .flatMapSingle(download -> moPubAdsManager.shouldHaveInterstitialAds()
             .flatMap(hasAds -> {
-              if (hasAds) {
+              if (hasAds && !isAppcUpdate) {
```

Comment:
Should make sure this is what we want. Sorry if we already discussed but this means that all migrator downloads in apps events will be sent with NO_ADS on it, which doesn't reflect the true state of it. Is there a special reason for this?